app/views/candidate_interface/other_qualifications/details/_form.html.erb

```diff
@@ -3,6 +3,15 @@
     <%= f.govuk_collection_select :institution_country, select_country_options, :id, :name, label: { text: t('application_form.other_qualification.institution_country.label'), size: 'm' } %>
     <%= f.govuk_text_field :grade, label: { text: t('application_form.other_qualification.grade.optional_label'), size: 'm' }, width: 10 %>
     <%= f.govuk_text_field :award_year, label: { text: t('application_form.other_qualification.award_year.label'), size: 'm' }, hint: { text: t('application_form.other_qualification.award_year.hint_text') }, width: 4 %>
+  <% elsif @form.btec? %>
+    <%= f.govuk_text_field :subject, label: { text: t('application_form.other_qualification.subject.label'), size: 'm' } %>
+    <%= tag.div(id: 'subject-autocomplete-data', data: { source: subject_autocomplete_data }) if subject_autocomplete_data %>
+    <%= f.govuk_radio_buttons_fieldset :grade, legend: { text: t('application_form.other_qualification.grade.label'), tag: 'span' } do %>
+      <% OTHER_UK_QUALIFICATION_GRADES.each do |grade| %>
+        <%= f.govuk_radio_button :grade, grade, label: { text: grade.to_s }%>
```

Comment:
Just a small nit from me. With govuk_radio_buttons you need to set `link_errors: true` on the first radio button in the collection. 

 <%= f.govuk_radio_button :grade, grade, link_errors: true, label: { text: grade.to_s }%>

This will still work with the iteration and will direct the user to the first radio button

![image](https://user-images.githubusercontent.com/42515961/101645696-91906c80-3a2e-11eb-9dec-76af53f4eb82.png)